core/script_debugger_remote.h

```diff
@@ -176,6 +178,8 @@ class ScriptDebuggerRemote : public ScriptDebugger {
 	virtual void profiling_end();
 	virtual void profiling_set_frame_times(float p_frame_time, float p_idle_time, float p_physics_time, float p_physics_frame_time);
 
+	virtual void set_skip_breakpoints(bool skip_breakpoints);
```

Comment:
Add `p_` prefix to the parameter name here.

Reply:
fixed

Reply:
This is not fixed yet.

Reply:
Not that occurrence :)

Reply:
it is now :)